<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>491</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:rgb(215, 174, 225);
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:rgb(0, 85, 127);
	
	font: 87 18pt &quot;Segoe UI Black&quot;;


}</string>
     </property>
     <property name="text">
      <string>Video Downloader</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:rgb(0, 100, 100);
	
	font: 87 11pt &quot;Segoe UI Black&quot;;


}</string>
     </property>
     <property name="text">
      <string>Only MP4 Downloader</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 7pt &quot;Lucida Fax&quot;;
	border-width:3px;
	border-radius:10px;
	border-color:red;
	border-style:solid;
	
} </string>
     </property>
     <property name="placeholderText">
      <string>URL Link</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:rgb(0, 100, 100);
	
	font: 87 11pt &quot;Segoe UI Black&quot;;


}</string>
     </property>
     <property name="text">
      <string>Paste Link here:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:rgb(0, 100, 100);
	
	font: 87 11pt &quot;Segoe UI Black&quot;;


}</string>
     </property>
     <property name="text">
      <string>Save To:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 7pt &quot;Lucida Fax&quot;;
	border-width:3px;
	border-radius:10px;
	border-color:red;
	border-style:solid;
	
} </string>
     </property>
     <property name="placeholderText">
      <string>Select Folder...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
	background-color:rgb(215, 174, 225);

}
QRadioButton::hover {
	
	color:red;
	

}</string>
     </property>
     <property name="text">
      <string>MP4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
	background-color:rgb(215, 174, 225);

}
QRadioButton::hover {
	
	color:red;
	

}</string>
     </property>
     <property name="text">
      <string>MP3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>370</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color:rgb(200, 200, 100);
	border-radius:10px;
	border-style:solid;
	border-width:3px;
	border-color:rgb(170, 85, 0);
	
	font: 10pt &quot;Ravie&quot;;
}

QPushButton::hover {
	background-color:rgb(170, 185, 255);

}
QPushButton::pressed {
	background-color:rgb(170, 185, 255);
	border-width:5px;
	font: 9pt &quot;Ravie&quot;;

}</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>250</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>290</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:rgb(0, 85, 127);
	
	font: 87 12pt &quot;Segoe UI Black&quot;;


}</string>
     </property>
     <property name="text">
      <string>Download in progress:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>150</y>
       <width>311</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:rgb(0, 170, 0);
	
	font: 87 18pt &quot;Matura MT Script Capitals&quot;;
	
	


}</string>
     </property>
     <property name="text">
      <string>Ahmadwali Rahmani</string>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>lineEdit</zorder>
    <zorder>label_4</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>radioButton</zorder>
    <zorder>radioButton_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>progressBar</zorder>
    <zorder>label_6</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
